config.toml

```diff
@@ -153,39 +153,34 @@ googleAnalytics = ""
 	[params.team]
         enable = true
 		title = "Our amazing team"
-		subtitle = "Lorem ipsum dolor sit amet consectetur."
-		description = "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde."
+		subtitle = "Changing the simulation world from a little ski town."
```

Comment:
I might use the tag line Changing the simulation world from empirically driven to simulation drive. (we can find the exact text in the slide)